src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -410,9 +404,10 @@ function PolicyAccountingPage({policy}: PolicyAccountingPageProps) {
                 errorText: synchronizationError,
                 errorTextStyle: [styles.mt5],
                 shouldShowRedDotIndicator: true,
-                description: isSyncInProgress
-                    ? translate('workspace.accounting.connections.syncStageName', connectionSyncProgress.stageInProgress)
-                    : translate('workspace.accounting.lastSync', datetimeToRelative),
+                description:
+                    isSyncInProgress && connectionSyncProgress?.stageInProgress
```

Comment:
Do we need this additional condition here? `&& connectionSyncProgress?.stageInProgress`

I believe this is checked in the `isConnectionInProgress` function and if it's not maybe we should check it there 🤔

Reply:
Yeah, but without this check, the below line will throw the warning that `connectionSyncProgress.stageInProgress` is possibly undefined.

```javascript
translate('workspace.accounting.connections.syncStageName', connectionSyncProgress.stageInProgress)
```

Reply:
Oh, right... So it's fine I guess 👍